triplea_maps.yaml

```
@@ -1924,6 +1924,14 @@
     <br>
     House of Habsburg Beta Version
     <br>Large map of Europe set in the 1500s
+- mapName: CrazyG's Big World War One
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/crazygs_ww1/archive/master.zip
+  version: 1
+  img: 
```

@CrazyGerman The build system rejected this entry because a thumbnail image is required.  Please provide an URL to a thumbnail image, preferably hosted in the _/description_ folder of the map repo.

---

Thanks for the response. The guide on the forum doesn't mention requiring an image.

Does this work?
```suggestion
  img: https://github.com/triplea-maps/crazygs_ww1/tree/master/description/thumbnail.jpg
```

---

Sorry, I should have clarified that the thumbnail image URL is required _only_ when you specify the `img` property.  That is, you can't just leave it blank.  However, you can remove the `img` line altogether if you don't want to specify a thumbnail.

If you do want to include the thumbnail you added to your repo, the correct URL would be

https://raw.githubusercontent.com/triplea-maps/crazygs_ww1/master/description

---

```suggestion
```
Ok, can I upload just without an image then? I don't have a good one prepared.

---

Yes, your proposed change should be fine.